CHANGELOG.rst

```diff
@@ -5,6 +5,14 @@ Kubernetes Collection Release Notes
 .. contents:: Topics
 
 
+v2.4.1
+======
+
+Release Summary
+---------------
+
+The kubernetes.core 2.4.1 release includes several trivial bug fixes related to sanity.
```

Comment:
```suggestion
The kubernetes.core 2.4.1 release includes several trivial bug fixes related to code sanity.
```
Minor nit: Some users may not be familiar with this term, I would suggest either using the full term "code sanity" or saying something like "code formatting" to avoid confusion.